src/main/java/picard/fingerprint/HaplotypeProbabilitiesFromGenotype.java

```diff
@@ -28,10 +28,17 @@
  * Represents a set of HaplotypeProbabilities that were derived from a single SNP
  * genotype at a point in time.
  */
-public class HaplotypeProbabilitiesFromGenotype extends HaplotypeProbabilities {
+public class HaplotypeProbabilitiesFromGenotype extends HaplotypeProbabilities implements Cloneable{
     private final Snp snp;
     private final double[] likelihoods;
 
+    @Override
+    public Object clone() throws CloneNotSupportedException {
+        HaplotypeProbabilitiesFromGenotype c = (HaplotypeProbabilitiesFromGenotype) super.clone();
+        System.arraycopy(likelihoods,0,c.likelihoods,0,likelihoods.length);
```

Comment:
more whitespace, please
```suggestion
        System.arraycopy(likelihoods, 0, c.likelihoods, 0, likelihoods.length);
```